docs/reference/content/builders/aggregation.md

```diff
@@ -142,6 +142,19 @@ This example outputs, for each document, a document for each element in the `siz
 unwind("$sizes")
 ```
 
+Available with MongoDB 3.2, this example also includes any documents that have missing or `null` values for the `$sizes` field or where 
+the `$sizes` list is empty:
+
+```java
+unwind("$sizes", new UnwindOptions().preserveNullAndEmptyArrays(true))
+```
+
+Available with MongoDB 3.2, this example unwinds the `sizes` array and also outputs the array index into the `$position` field:
```

Comment:
Oh, I see.  You don't actually need to $-prefix the index field name.  In that case, let's remove the $, as to me it implied that the actual field name would be 'position' rather than '$position', given the way '$' has to be used elsewhere in agg pipelines.